tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -853,6 +853,70 @@ abstract class BaseWorksheetLspSuite(
     } yield ()
   }
 
+  if (!ScalaVersions.isScala3Version(scalaVersion))
+    test("using-completion") {
+      for {
+        _ <- initialize(
+          s"""
+             |/metals.json
+             |{
+             |  "a": {
+             |    "scalaVersion": "${scalaVersion}"
+             |  }
+             |}
+             |/a/src/main/scala/foo/Main.worksheet.sc
+             |//> using dep io.cir
+             |//> using dep io.circe::circe-ref
+             |//> using dep com.lihaoyi::upickle:1.4
+             |""".stripMargin
+        )
+        _ <- server.didOpen("a/src/main/scala/foo/Main.worksheet.sc")
+        groupExpectedCompletionList =
+          """|io.circe
+             |io.circul""".stripMargin
+        groupCompletionList <- server.completion(
+          "a/src/main/scala/foo/Main.worksheet.sc",
+          "//> using dep io.cir@@",
+        )
+        _ = assertNoDiff(groupCompletionList, groupExpectedCompletionList)
```

Comment:
I prefer the expected value inlined, it's easier to read, but that's only an opinion.

Reply:
done